Makefile

```diff
@@ -239,6 +240,15 @@ flexdll_initer_gnat.o: flexdll_initer.c
 flexdll_initer_mingw64.o: flexdll_initer.c
 	$(MIN64CC) $(GCC_FLAGS) -c -o $@ $<
 
+build_config.ml: Makefile
+	echo 'let mingw_prefix = "$(MINGW_PREFIX)"' > $@
+	echo 'let mingw64_prefix = "$(MINGW64_PREFIX)"' >> $@
+	echo 'let msvc = "$(subst ",\",$(subst \,\\,$(MSVCC)))"' >> $@
+	echo 'let msvc64 = "$(subst ",\",$(subst \,\\,$(MSVCC64)))"' >> $@
+	echo 'let cygwin64 = "$(subst ",\",$(subst \,\\,$(CYG64CC)))"' >> $@
+	echo 'let mingw = "$(subst ",\",$(subst \,\\,$(MINCC)))"' >> $@
+	echo 'let mingw64 = "$(subst ",\",$(subst \,\\,$(MIN64CC)))"' >> $@
```

Comment:
I think in practice these should never end up with anything other than a compiler name, so perhaps the `subst` can be omitted?